baywatch.info.yml

```diff
@@ -23,6 +23,7 @@ dependencies:
   - smtp:smtp
   - config_split:config_split
   - jwt:jwt
+  - bay_platform_dependencies:bay_platform_dependencies
```

Comment:
Do we need the call to `$this->baywatch_install_module('bay_platform_dependencies');` when the dependency is added here?

Reply:
[This one](https://github.com/dpc-sdp/baywatch/pull/79/files#diff-15f33d16d9cd8e8b1e20709068a9414d6a238b1a6e9d2f066dee5250a0dd1abbR46)?

Reply:
Yeah you do for existing installs.